webpack.config.js

```diff
@@ -51,6 +55,20 @@ function getAliasesForExtensions() {
 	return aliasesMap;
 }
 
+/*
+ * Find package.json for file at `filepath`.
+ * Return `true` if it has a property whose key is `PROPKEY_ESNEXT` or 'module'.
+ */
+function hasPkgEsnext( filepath ) {
+	const pkgRoot = findRoot( filepath );
+	const packageJsonPath = path.resolve( pkgRoot, 'package.json' );
+	const packageJsonText = fs.readFileSync( packageJsonPath,
+		{ encoding: 'utf-8' } );
+	const packageJson = JSON.parse( packageJsonText );
```

Comment:
Will this never throw?

Reply:
if it doesn't the build should stop - what would we do otherwise?

Reply:
Just pondering why we were guarding so much a line below 😄

Reply:
Hmm, good thinking. OTOH, all those `package.json`s are also used by npm, so I'd hope they're well-formed...